tools/dm-task-configuration-file-intro.md

```diff
@@ -0,0 +1,183 @@
+        black-white-list:  "instance"                                      # The black white list item selected from `black-white-list` above 
+
+        mydumper-config-name: "global"                                     # The mydumper configuration name. You cannot set it 
+                                                                           # and `mydumper` at the same time. 
+        loader-config-name: "global"                                       # The Loader configuration name. You cannot set it and
+                                                                           # `loader` at the same time.
+        syncer-config-name: "global"                                       # The Syncer configuration name. You cannot set it and 
+                                                                           # `syncer` at the same time.
+
+    -
+        config:
+            host: "192.168.199.118"
+            port: 5306
+            user: "root"
+            password: "1234"
+        instance-id: "instance118-5306"
+
+        mydumper:                                                          # The mydumper configuration. You cannot set it and 
+                                                                           # `mydumper-config-name` at the same time.
+            mydumper-path: "./mydumper"                                    # The mydumper binary file path. It is generated by 
+                                                                           # Ansible deployment application and needs no 
+                                                                           # configuration.
+            threads: 4
+            chunk-filesize: 8
+            skip-tz-utc: true
+            extra-args: "-B test -T t1,t2"
+    
+        loader:                                                            # The Loader configuration. You cannot set it and 
+                                                                           # `loader-config-name` at the same time.
+            pool-size: 32                                                  # The number of threads that execute mydumper SQL 
+                                                                           # files concurrently in Loader
+            dir: "./dumped_data"
+    
+        syncer:                                                            # The Syncer configuration. You cannot set it and 
+                                                                           # `syncer-config-name` at the same time.
+            worker-count: 32                                               # The number of threads that synchronize binlog events 
+                                                                           # concurrently in Syncer
+            batch: 2000
+            max-retry: 200
+```
+
+Refer to [Feature configuration set](#feature-configuration-set) to get the configuration details of the following options:
+
+- `route-rules`
```

Comment:
To make it clearer, (for these options in the list) please specify which option the users should refer to in "Feature configuration set".  Ref: https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/configuration/configuration.md